src/compile-middleware.ts

```diff
@@ -75,35 +71,33 @@ export const babelCompileCache = LRU<string>(<LRU.Options<string>>{
 
 
 export function babelCompile(forceCompile: boolean): RequestHandler {
-  if (forceCompile == null) {
-    forceCompile = false;
-  }
-
   return transformResponse({
-    shouldTransform(_request: Request, response: Response) {
-      return isSuccessful(response) &&
-          compileMimeTypes.indexOf(getContentType(response)) >= 0;
+    shouldTransform(request: Request, response: Response) {
+      // We must never compile the Custom Elements ES5 Adapter or other
+      // polyfills/shims.
+      const isPolyfill = url.parse(request.url)
+                             .pathname.split('/')
+                             .indexOf('webcomponentsjs') >= 0;
```

Comment:
Here's what the CLI does: https://github.com/Polymer/polymer-cli/blob/master/src/build/optimize-streams.ts#L73

Same idea here, although the CLI is looking at a file path (hence the check for two types of delimiters) while here we're looking at a URL.

Could we instead check specifically that the file name is `custom-elements-es5-adapter.js`? I wasn't sure if we need to be careful about not compiling anything from the entire package (which is what CLI seems to be assuming), or if it's really only this specific file that matters.